src/In.ProjectEKA.HipService/wwwroot/openapi.yaml

```diff
@@ -284,6 +284,62 @@ paths:
             application/xml:
               schema:
                 $ref: '#/components/schemas/ErrorResponse'
+  /health-information/{informationId}:
+    post:
+      tags:
+        - data flow
+      summary: Health information link data request
+
+      description: >
+        Return data respective to the link
+      parameters:
+        - $ref: '#/components/parameters/InformationId'
+        - $ref: '#/components/parameters/Token'
+        - $ref: '#/components/parameters/TransactionId'
```

Comment:
Why do we need transaction Id if we have information id already?

Reply:
If we want to identify for which data flow transaction that the link data is being fetched, we can have transactionId

Reply:
information id is unique right, we could back track from that always right.

Reply:
You are right. Logically, we are not doing anything with the transactionId in this api. we are just returning it back in the response. So, we can remove the transactionId in this api I guess.